tests/pubsub/test_floodsub.py

```diff
@@ -45,68 +44,15 @@
 
     node_a_id = str(node_a.get_id())
 
-    msg = MessageTalk(node_a_id, node_a_id, ["my_topic"], "some data", generate_message_id())
-
-    await floodsub_a.publish(node_a.get_id(), msg.to_str())
-
+    msg = generate_RPC_packet(node_a_id, ["my_topic"], "some data", generate_message_id())
+    await floodsub_a.publish(node_a_id, msg.SerializeToString())
     await asyncio.sleep(0.25)
 
     res_b = await qb.get()
 
     # Check that the msg received by node_b is the same
     # as the message sent by node_a
-    assert res_b == msg.to_str()
-
-    # Success, terminate pending tasks.
-    await cleanup()
-
-@pytest.mark.asyncio
-async def test_simple_three_nodes():
```

Comment:
Just documenting, why did we remove this test? Too repetitive?

Reply:
@robzajac  yes, this test is covered by `test_three_nodes_two_topics_test_obj`